app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -338,6 +339,7 @@ private object SparkFSFetcher {
 
   var defEventLogDir = "/system/spark-history"
   var defEventLogSizeInMb = 100d; // 100MB
+  var defSparkLogExt = "_1.snappy"
```

Comment:
Anyway, I don't think this has to be addressed here though... `.snappy` vs `.[other format]` themselves also worth a configuration.